mlshvch/hometask-5/8.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE film_actor
+DROP CONSTRAINT film_actor_film_id_fkey;
+
+ALTER TABLE film_category
+DROP CONSTRAINT film_category_film_id_fkey;
+
+ALTER TABLE inventory
+DROP CONSTRAINT inventory_film_id_fkey;
```

Comment:
Why do you need to drop constraints? This task does not involve changing DB structure.

Reply:
Deletion is blocked because of foreign keys

Reply:
Foreign keys exist to maintain database integrity and doing their job of preventing you from deleting records and leaving the dependent records pointing to the nonexistent ones. I recommend to get back to the PostgreSQL docs and pay attention to DELETE command options, the answer is there.

Reply:
Do you mean ONLY option?

Reply:
No, I mean CASCADE option, but somehow I can't find it in the docs

Reply:
Honestly, I had a thought about cascade, but wasn't sure

Reply:
FYI: there is no cascade option, that's right. You should delete other references before actually deleting the films.

Reply:
But at this moment I don't understand how to use it: this option has to be added during DB design, so on `DELETE` command everything has to happen automatically without any additional parameters

Reply:
Yes, please see the my previous comment. 😉

Reply:
Sure) Today something with Internet or GitHub on my side: delays happen frequently